### ISSUE TITLE
LineChart: lines missing, tooltip error

### ISSUE THREAD

**Issue by LearnShare** [LearnShare/d3-chart#4]
![image](https://cloud.githubusercontent.com/assets/3265208/10809103/0fb85f5e-7e2e-11e5-961a-e25aed371fe7.png)
![image](https://cloud.githubusercontent.com/assets/3265208/10809124/5d500b40-7e2e-11e5-9ad1-9f426d072083.png)

**Comment by LearnShare** [comment]
Error:
![image](https://cloud.githubusercontent.com/assets/3265208/10809315/8d489f72-7e30-11e5-84f8-358d7f0814c9.png)

x value from:

``` js
$scope.chartDatas = [
  [{
    x: new Date('2015-06-10 16:27:00'),
    y: 27
  }, {
    x: new Date('2015-06-11 16:28:00'),
    y: 16
  }]
];
```

Fix time format:

``` js
$scope.chartDatas = [
  [{
    x: new Date('2015-06-10T16:27:00+08:00'),
    y: 27
  }, {
    x: new Date('2015-06-11T16:28:00+08:00'),
    y: 16
  }]
];
```

**Comment by LearnShare** [comment]
Fixed in [381d7f65](https://github.com/LearnShare/d3-chart/commit/381d7f652c39971ac898c070f690fbb27cd99441).
